src/sentry/static/sentry/app/components/events/meta/annotatedText.jsx

```diff
@@ -0,0 +1,137 @@
+import _ from 'lodash';
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'react-emotion';
+
+import InlineSvg from 'app/components/inlineSvg';
+import Tooltip from 'app/components/tooltip';
+import {t, tn} from 'app/locale';
+import utils from 'app/utils';
+
+const Chunks = styled.span`
+  span {
+    display: inline;
+  }
+`;
+
+const Redaction = styled.span`
+  background: rgba(255, 0, 0, 0.05);
+  cursor: default;
+`;
+
+const Placeholder = styled(Redaction)`
+  background: rgba(255, 0, 0, 0.05);
+  cursor: default;
```

Comment:
You overwrite all the styles defined in `Redaction`, this could be based on `styled.span` instead.

Reply:
Thanks, those two lines were actually meant to go away by inheriting `Redaction`!